project/LanceWang12/readme.md

```diff
@@ -0,0 +1,102 @@
+# t-SNE implementation
+
+
+
+## Basic Information
+
+t-SNE (t-distributed stochastic neighbor embedding) is an dimension reduction algorithm.  It can be used in visualizing high dimension  data and feature reduction.  **By the way, I will compare my implementation with sklearn.**
+
+- github repository: https://github.com/LanceWang12/t-SNE-implementation
+
+
+
+## Problem to solve
+
+In the Big Data analysis, data is not only becoming bigger and bigger; but also becoming more and more complicated.  It causes a spectacular increase of the dimensionality of the data.  For example, the dimensionality of a set of images is the number of pixels in any image, which ranges from thousands to millions.
+
+Computer has no problem processing that many dimensions.  However, we are limited to three dimensions.  Computer still needs us, so we should find a way to effectively visualize high-dimensional data before handing it over to the computer.
```

Comment:
Visualization is a key to presenting the results of numerical calculations and the topic is well aligned with the course.